components/List/list.html

```diff
@@ -0,0 +1,190 @@
+                            <a href="/components/Avatar/avatar.html">
+                                Avatar
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Alert/alert.html">
+                                Alert
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/Components/Badge/badge.html">
+                                Badge
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Button/button.html">
+                                Button
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Card/card.html">
+                                Card
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Image/image.html">
+                                Image
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/Components/Input/input.html">
+                                Input
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Navigation/navigation.html">
+                                Navigation
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Rating/rating.html">
+                                Rating
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="/components/Toast/toast.html">
+                                Toast
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Modal/modal.html">
+                                Modal
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/List/list.html">
+                                List
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="doc-content">
+
+                <section class="Lists ">
+                    <h4 class="mb-3 text-primary-color">Lists</h4>
+                    <p class="mb-5 lh-8">Lists are used to group related content together and displays collection of
+                        information vertically or horizontally.</p>
+                    <p class="mb-3 "> <b>Stacked List</b> : list component where the items are stacked vertically
+                        without any bullets.
+
```

Comment:
I suggest using beautify/prettier to remove all these extra spaces and indentations.